lib/pyenv.rb

```diff
@@ -18,16 +18,18 @@ def initialize(build_wrapper)
 
     def setup!
       install!
+      update!
```

Comment:
You're right! I did not notice the reason for update was to make sure we have latest pyenv so that the list of available versions to install is "refreshed", this makes total sense of course. Fixing now.